hi-ml-azure/src/health_azure/himl.py

```diff
@@ -824,7 +818,10 @@ def submit_to_azure_if_needed(  # type: ignore
         will also register the data in this folder as an AzureML dataset.
     :param output_datasets: The script will create a temporary folder when running in AzureML, and while the job writes
         data to that folder, upload it to blob storage, in the data store.
-    :param num_nodes: The number of nodes to use in distributed training on AzureML.
+    :param num_nodes: The number of nodes to use in distributed training on AzureML. When using a value > 1, an
```

Comment:
"When using a value > 1, an distributed MPI job will be started" <- this is only true if pytorch_processes_per_node_v2=None

Also, what about the third case where num_nodes=1 and pytorch_processes_per_node_v2=None - is this considered a PyTorch job? Is it compatible with PyTorch Lightning?